### DISCUSSION TITLE
impossible to install apicurio studio

### DISCUSSION THREAD

**Discussion by jmonsel** [ApicurioArchive/apicurio-studio#2558] [General]
Hello, Sorry if this is not the right place to share my problem... 
I'm trying to install apicurio studio for POC and learning activities but I've got 2 problems : 
- Download button from Apicurio studio web page lead to a 404 page - https://www.apicur.io/studio/
- So I download asset (0.2.62 final-quickstart zip) from github release page (https://github.com/Apicurio/apicurio-studio/releases) but when I unzip the archive I've got errors on lots of files saying "access path too long"....

Any help or advice ?

thank you in advance

**Comment by EricWittmann** [comment]
My bad on the 404 error!  I'll get that fixed.

**Comment by EricWittmann** [comment]
Okay, I think I fixed the website. Although you could also click on getting started in the left hand nav. 

The most recent beta version of studio does require docker.

**Reply by EricWittmann** [reply]
The older version's getting started download zip file does have some long file paths inside of it. I'm afraid there isn't anything I can do about that. 

Depending on your system and perhaps the options available to you for your unzip utility, your mileage may vary. 😥
